packages/pilot/src/pages/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import {
+  applyTo,
+  both,
+  compose,
+  complement,
+  equals,
+  evolve,
+  find,
+  identity,
+  ifElse,
+  is,
+  isNil,
+  pipe,
+  prop,
+  propEq,
+  propSatisfies,
+  uncurryN,
+} from 'ramda'
+import { withRouter } from 'react-router-dom'
+import { translate } from 'react-i18next'
+
+import { receiveReactError } from '.'
+
+const mapStateToProps = ({ errors }) => ({ errors })
+
+const mapDispatchToProps = dispatch => ({
+  receiveError: (error, stack) => {
+    dispatch(receiveReactError({ error, stack }))
+  },
+})
+
+const enhanced = compose(
+  connect(
+    mapStateToProps,
+    mapDispatchToProps
+  ),
+  withRouter,
+  translate()
+)
+
+const findInErrorsRoutes = uncurryN(2, route => find(pipe(
+  prop('affectedRoute'),
+  equals(route)
+)))
+
+const isObject = both(is(Object), complement(isNil))
+
+const localizeError = uncurryN(2, t => ifElse(
+  both(isObject, propSatisfies(complement(isNil), 'localized')),
+  evolve({
+    localized: applyTo(t),
+  }),
+  identity
+))
+
+class ErrorBoundary extends Component {
+  constructor () {
+    super()
+
+    this.cloneChild = this.cloneChild.bind(this)
+    this.handleError = this.handleError.bind(this)
+  }
+
+  componentDidCatch (error) {
+    this.props.receiveError(error)
+  }
+
+  handleError (error, affectedRoutes = []) {
+    const { location: { pathname } } = this.props
+
+    this.props.receiveError({
+      affectedRoutes: [...affectedRoutes, pathname],
+      error,
+    })
+  }
+
+  cloneChild (child) {
+    const {
+      errors,
+      location: { pathname },
+      t,
+    } = this.props
+
+    const filteredError = localizeError(
```

Comment:
Aqui pode ficar tudo em uma linha só pq são exatos 80 caracteres 😄 
```suggestion
    const filteredError = localizeError(t, findInErrorsRoutes(pathname, errors))
```